paper-stepper.html

```diff
@@ -13,6 +13,7 @@
 <link rel="import" href="animations/fade-in-slide-from-left-animation.html">
 <link rel="import" href="animations/fade-out-slide-left-animation.html">
 <link rel="import" href="paper-step.html">
+<link rel="import" href="../bower_components/neon-animation/web-animation.html">
```

Comment:
Replace: 
`<link rel="import" href="../bower_components/neon-animation/web-animation.html">`
with:
`<link rel="import" href="../neon-animation/web-animations.html">`
It complains that the path doesn't exist and it also lacks the s at the end of animations.